cs-config/install.sh

```diff
@@ -0,0 +1,2 @@
+conda env create -f environment.yml
+conda activate calc
```

Comment:
@MaxGhenis I think we are going to want to do something like this:

```
conda update env --name base -f environment.yml
```

Docker and conda environments don't really play well together. You have to do a [good bit of magic](https://pythonspeed.com/articles/activate-conda-dockerfile/) to get them to work properly. 

This will install the packages into the base environment which is used when running the app.